.circleci/config.yml

```diff
@@ -28,8 +28,8 @@ jobs:
             yarn
             yarn dist
             cd applications/desktop/dist/
-            chmod a+x nteract-0.11.6-x86_64.AppImage
-            ./nteract-0.11.6-x86_64.AppImage --appimage-extract
+            chmod a+x nteract-0.11.7-x86_64.AppImage
+            ./nteract-0.11.7-x86_64.AppImage --appimage-extract
```

Comment:
It might be worth it to make this whole flow here one bash script (or node.js script), because then we can detect what the built image is by name and then re-use it here.

Reply:
OK, can do 👍